db/schema.rb

```diff
@@ -17,6 +17,7 @@
     t.string   "first_name", null: false
     t.string   "last_name",  null: false
     t.date     "dob",        null: false
+    t.boolean "estimated_dob", null: false
```

Comment:
You shouldn't really be modifying this file, it's generated. You should be working through migrations.